apps/showcase/tsconfig.app.json

```diff
@@ -26,8 +26,8 @@
     "module": "ES2022",
     "useDefineForClassFields": false,
     "noEmitHelpers": true,
-    "noUnusedLocals": true,
-    "noUnusedParameters": true,
+    "noUnusedLocals": false,
```

Comment:
Why removing this restriction?

Reply:
no longer needed, the generated SDK had a lot of these errors but I fixed them anyway